### ISSUE TITLE
Volume useDecibel parameter does not change format

### ISSUE THREAD

**Issue by Kruiser8** [Palakis/obs-websocket#513]
##### Issue type
Bug report

##### Description
GetVolume and SetVolume do not use the decibel format when the `useDecibel` parameter is supplied. 

##### Steps to reproduce and other useful info
I'm using the obs-websocket-js library and running the following:
```js
obs = new OBSWebSocket();
obs.connect({ address: 'localhost:4444', password: '...' })
  .then(async () =>{
    return await obs.send('GetVolume', {
      'source': 'Desktop Audio',
      'useDecibel': true
    });
  }).then((data) =>{
    console.log(data);
  }).catch(err => {
    console.error(JSON.stringify(err));
  });
```
Regardless of the `useDecibel` value, I receive the volume in mul format: `volume: 0.4000000059604645`. 

Similarly, `SetVolume` only accepts volume values between 0 and 1. If I pass in the following:
```js
obs.send('SetVolume', {
  'source': 'Desktop Audio',
  'volume': -20,
  'useDecibel': true
});
```
Then, I receive the following error:
`{"error":"invalid request parameters","message-id":"2","status":"error","messageId":"2"}`

##### Technical information
- **Operating System** : Win 10
- **OBS Studio version** : 24.0.3
- **OBS Websocket version** : 4.7.0

**Comment by Kruiser8** [comment]
OBS Log file: [2020-05-17 15-42-29.txt](https://github.com/Palakis/obs-websocket/files/4640901/2020-05-17.15-42-29_abridged.txt)

**Comment by tt2468** [comment]
This is because you are still on version 4.7.0, which does not support the recently added `useDecibel` parameter. You will need to update to OBS version > 25.0.7, and use the attached release candidate of the plugin.
[windows_build.zip](https://github.com/Palakis/obs-websocket/files/4640910/windows_build.zip)

**Comment by Kruiser8** [comment]
Thanks! Good to know. I was looking at the 4.x-current version of the protocol (4.8) which is actually ahead of the current release (4.7). My bad, I'll switch to looking at the protocol.md included with the 4.7 release to avoid getting ahead.
